### ISSUE TITLE
links to Stencils and Templates are broken

### ISSUE THREAD

**Issue by BrunoBieri** [evolus/pencil#579]
https://pencil.evolus.vn/Stencils-Templates.html

![Stencils](https://user-images.githubusercontent.com/16573838/67260782-8cc0d000-f49d-11e9-8efe-511dc1666c8d.png)

All the links to the stencil templates are broken.

**Comment by Aeyk** [comment]
Good point. The links in the Google Code page are still valid, I'll include them below. 

Does anyone know the process for changing the website's links? 

I tried to reach out to the author of some old commits. Fingers crossed.

Android 4.0 (ICS)
https://storage.googleapis.com/google-code-archive-downloads/v2/code.google.com/evoluspencil/Android.GUI_v2.zip

iOS wireframe:
https://storage.googleapis.com/google-code-archive-downloads/v2/code.google.com/evoluspencil/iOS-Wireframe_v2.zip

iOS GUI:
https://storage.googleapis.com/google-code-archive-downloads/v2/code.google.com/evoluspencil/iOS.GUI_v2.zip

ExtJS (v14):
https://storage.googleapis.com/google-code-archive-downloads/v2/code.google.com/evoluspencil/ExtJs.GUI_v14.zip

Dojo (v6):
https://storage.googleapis.com/google-code-archive-downloads/v2/code.google.com/evoluspencil/Dojo.GUI_v6.zip
